### PR TITLE
docs(*): copyedit readme, intro

### PR DESCRIPTION
**Pull request by jo3-l**
**Please describe the changes this PR makes and why it should be merged:**

**Status**

- [x] Code changes have been tested on an instance of YAGPDB, or there are no code changes
- [x] I have read and followed the [contribution guide](https://github.com/yagpdb-cc/yagpdb-cc/blob/master/CONTRIBUTING.md)
- [x] I have [written documentation](https://github.com/yagpdb-cc/yagpdb-cc/blob/master/WRITING-DOCUMENTATION.md) for my changes, or there is no need to

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,44 +2,44 @@
 
 <h1 align="center"><img src="https://yagpdb.xyz/static/img/logo_y.png" height=32px width=32px></img>&nbspYAGPDB Custom Commands</h1>
 
-> <p align="center">A curated collection of CCs for your server</p>
+> <p align="center">A curated collection of custom commands for your server</p>
 
 <p align="center">
 	<i>
-		If you are not planning on contributing, but rather simply viewing custom commands, please use our <a href="https://yagpdb-cc.github.io">website</a> instead, which displays all custom commands available here in a user-friendly manner!
+		If you are looking to browse through our collection of custom commands as opposed to contributing, please refer to our <a href="https://yagpdb-cc.github.io">website</a> instead, which displays all custom commands available in a user-friendly manner.
```
Comment (l-zeuch) [inline README.md:9]:
```suggestion
		If you are looking to browse through our collection of custom commands, please refer to our <a href="https://yagpdb-cc.github.io">website</a> instead, which displays all custom commands available in a user-friendly manner.
```

I think "just browsing" implies "not looking to contribute", so this feels a bit redundant.

File: README.md [hunk 2]
```diff
@@ -2,44 +2,44 @@
...
+- **Extensive:** There are over sixty custom commands (and we're adding more regularly!), meaning that it's likely that you'll be able to find what you're looking for.
 - **Official:** This collection of commands is [maintained by a group of YAGPDB staff members](#contributors) and has official support.
-- **Well tested:** We review all commands added and many commands have widespread usage, meaning that it's unlikely - [though not impossible](#disclaimer) - for there to be serious bugs in them.
-- **Well documented:** We take pride in our high quality of documentation. It should be _easy_ for someone to figure out how to use any command here given that they are willing to read and have common sense.
+- **Well tested:** We review all commands added and many commands have widespread usage, meaning that it's unlikely — [though not impossible](#disclaimer) — for there to be serious bugs in them.
+- **Well documented:** We take pride in our high quality of documentation, which makes it easy for anyone to figure out how to use a given command provided that they are willing to read.
```
Comment (l-zeuch) [inline README.md:20]:
```suggestion
- **Well documented:** We take pride in our high quality of documentation, which makes it easy for anyone to figure out how to use a given command.
```

File: README.md [hunk 3]
```diff
@@ -2,44 +2,44 @@
...
 
 # Disclaimer
 
 The YAGPDB developer, staff, and/or support are not responsible for any issues with or caused by these custom commands. Furthermore, though we thoroughly review all commands added, they are not guaranteed to be working; use them at your own risk.
 
 # Need help?
 
-If you're having issues with one of the custom commands here, please get in touch via the [YAGPDB Support Server](https://discord.com/invite/5uVyq2E), where we can help further. If you prefer, you can also use [Discussions](https://github.com/yagpdb-cc/yagpdb-cc/discussions).
+If you are experiencing issues with a custom command from this collection, please get in touch via the [YAGPDB Support Server](https://discord.com/invite/5uVyq2E), where we can provide further assistance. If you prefer, you can also [Open a GitHub Discussion](https://github.com/yagpdb-cc/yagpdb-cc/discussions).
```
Comment (l-zeuch) [inline README.md:28]: Personally I'd prefer it if we keep issues and requests regarding this repository on GitHub. The majority on the server does not know all the ins and outs of some random:tm: code from some repo, and here we can properly track bugs and suchlike.

Feel free to dismiss this comment though.

**Review by l-zeuch — CHANGES_REQUESTED** [review]
FTR I just commented on the `README.md` file, as the introduction on the website will get the same contents anyway.

File: README.md (hunk 3 above)
Comment (jo3-l) [reply README.md:28]: For trivial questions I think it still makes sense to suggest asking in the server.

Comment (l-zeuch) [reply README.md:28]: Definitely agree for trivial questions.

Though, for actual:tm: issues like "the command is giving an error" (not user-error), that should be tracked here.

I honestly have no idea how to phrase this, so I'd say we keep it as-is, unless you find a better way.

Comment (jo3-l) [reply README.md]: SGTM. I'll mull over this, but don't think this concern should block the PR.

Comment (l-zeuch) [reply README.md]: Yeah, let's keep it as-is.

**Review by l-zeuch — APPROVED** [review]
:shipit:

**Pull request merged**